src/api/app/controllers/webui/users/tokens_controller.rb

```diff
@@ -49,13 +49,14 @@ def create
     authorize @token
 
     respond_to do |format|
-      format.js do
+      format.html do
         if @token.save
-          flash.now[:success] = "Token successfully created! Make sure you save it - you won't be able to access it again."
-          render partial: 'create', locals: { string: @token.string }
+          flash[:success] = "Token successfully created! Make sure you save it - you won't be able to access it again."
+          flash[:show_token] = 'true'
```

Comment:
This flash is displayed to users when they create a token (see the `true` in the screenshot below):

![bug](https://user-images.githubusercontent.com/1102934/150326408-b94206f5-c9e0-4078-9a10-deb44017cb0e.png)

Using an instance variable instead of storing something in a flash message would be a better option here. This way, you can still check if this instance variable is set, then display the token string